src/components/ToolchainSection/ToolchainSection.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { Box, Grid } from 'grommet';
+
+import { Heading } from '../../legos/typography/Heading';
```

Comment:
Do you even use Heading component? My linter says that `'Heading' is defined but never used`